src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -1,15 +1,169 @@
 import { ArrowForwardIcon } from '@chakra-ui/icons'
-import { Button, HStack, Stack } from '@chakra-ui/react'
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  Skeleton,
+  SkeletonCircle,
+  Tag,
+  TagLabel
+} from '@chakra-ui/react'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { useMemo } from 'react'
 import { NavLink } from 'react-router-dom'
+import { Column } from 'react-table'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
-import { Text } from 'components/Text'
+import { ReactTable } from 'components/ReactTable/ReactTable'
+import { RawText, Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
 
-import { StakingOpportunitiesRow } from './StakingOpportunitiesRow'
+type StakingOpportunity = {
+  id: number
+  validatorName: string
+  apr: number
```

Comment:
The `apr` prop could technically be a number as it's nothing in the range of a BN but it will come as a string from chain-adapters (https://github.com/shapeshift/lib/pull/491/files#diff-e72c79aae6dd3ad84fe702284223d9f1ecee635105eec56322a54718fbc7310fR14) and transformed to a BN from the yet-to-be-built selectors so you can assume it's a BN